### PR TITLE
Redirect 404s -- Docs Cite

### PR DESCRIPTION
**Pull request by GranSpencer09**
@msh100

### CODE REVIEW

**Comment by GranSpencer09** [comment]
I'm not sure what caused the failure here

File: cloud/azure/azure.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Portworx on Azure"
 keywords: portworx, azure, PaaS, IaaS, docker, converged, cloud
 sidebar: home_sidebar
-redirect_from: "/portworx-on-azure.html"
+redirect_from:
+  - /portworx-on-azure.html
+  - /cloud/azure/azure.html
```
Comment (msh100) [inline cloud/azure/azure.md]: Here is the problem @GranSpencer09 - you're overwriting it's own page with a redirect to itself. This change is an issue.

**Comment by GranSpencer09** [comment]
Just made the changes, thank you very much Marcus I really appreciate it!

File: index.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ layout: page
 title: "Welcome to Portworx Docs"
 keywords: portworx, px-enterprise, px-developer, containers, storage
 sidebar: home_sidebar
+youtubeId : 0zTjOly0vkA
+redirect_from:
+  - /architecture.html
```
Comment (msh100) [inline index.md]: This causes a build failure. Also `architecture.md` seems like a page worth keeping.

@michaelferranti should the content of this be included in the home page?

**Comment by GranSpencer09** [comment]
@msh100

File: applications/hadoop-docker.md [hunk 3]
```diff
@@ -3,7 +3,8 @@ layout: page
 title: "A Production Ops Guide to Deploying Hadoop in Docker Containers"
 keywords: portworx, px-developer, hadoop, yarn, database, cluster, storage
 sidebar: home_sidebar
-
+redirect_from:
+  - /reference-architecture/sdn.html
```
Comment (msh100) [inline applications/hadoop-docker.md]: Why does an sdn link go to hadoop?

Check https://docs.portworx.com/knowledgebase/sdn.html maybe?

File: applications/mssql-server.md [hunk 4]
```diff
@@ -3,6 +3,8 @@ layout: page
 title: "SQL Server with Persistent Storage"
 keywords: portworx, container, microsoft, sqlserver, persistent storage, docker, kubernetes
 sidebar: home_sidebar
+redirect_from:
+  - /run_portworx_on_ddc.html
```
Comment (msh100) [inline applications/mssql-server.md]: Why from the "ddc" page (I assume docker datacenter)?

File: cloud/aws/ecs.md [hunk 5]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Amazon ECS with Portworx"
 keywords: portworx, amazon, docker, aws, ecs, cloud
 sidebar: home_sidebar
-redirect_from: "/portworx-on-ecs2.html"
+redirect_from:
+  - /portworx-on-ecs2.html
+  - /cloud/aws-ecs.html
```
Comment (msh100) [inline cloud/aws/ecs.md]: There's also a `portworx-on-ecs.md` file at the root of this repo, should this also get directed here?

File: getting-started/asap.md [hunk 6]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Get Started ASAP"
 keywords: portworx, AWS, CloudFormation
 sidebar: home_sidebar
-redirect_from: "/get-started-asap.html"
+redirect_from:
+  - /get-started-asap.html
+  - /get-started-px-enterprise.html
```
Comment (msh100) [inline getting-started/asap.md]: I feel like this belongs on `getting-started/px-enterprise.md`

File: install/docker.md [hunk 7]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Run PX with Docker"
 keywords: portworx, px-developer, px-enterprise, install, configure, container, storage, add nodes
 sidebar: home_sidebar
-redirect_from: "/run-with-docker-ent.html"
+redirect_from:
+  - /run-with-docker-ent.html
+  - /run-px-etcd-marathon.html
```
Comment (msh100) [inline install/docker.md]: Should this be redirecting elsewhere?

File: maintain/scale-up.md [hunk 8]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Adding storage to existing PX Cluster Nodes"
 keywords: scale-up
 sidebar: home_sidebar
-redirect_from: "/scale-up.html"
+redirect_from:
+  - /scale-up.html
+  - /convert-x86-to-storage-nodes.html
```
Comment (msh100) [inline maintain/scale-up.md]: What was this page? Does it belong here?

File: manage/encrypted-volumes.md [hunk 9]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Portworx Encrypted Volumes"
 keywords: portworx, px-developer, px-enterprise, plugin, install, configure, container, storage, encryption
 sidebar: home_sidebar
-redirect_from: "/encrypted-volumes.html"
+redirect_from:
+  - /encrypted-volumes.html
+  - /manage/encrypted-volumes.md
```
Comment (msh100) [inline manage/encrypted-volumes.md]: Redirect loop. Remove this.

File: release-notes/portworx-enterprise.md [hunk 10]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "PX-Enterprise Release notes"
 keywords: portworx, px-enterprise, release notes
 sidebar: home_sidebar
-redirect_from: "/px-enterprise-release-notes.html"
+redirect_from:
+  - /px-enterprise-release-notes.html
+  - /release-notes-1-0-4.html
```
Comment (msh100) [inline release-notes/portworx-enterprise.md]: Should we also delete all the release notes in the root of the repo in favour of this?

File: install/docker.md [hunk 11]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Run PX with Docker"
 keywords: portworx, px-developer, px-enterprise, install, configure, container, storage, add nodes
 sidebar: home_sidebar
-redirect_from: "/run-with-docker-ent.html"
+redirect_from:
+  - /run-with-docker-ent.html
```
Comment (msh100) [inline install/docker.md]: This still exists

File: scheduler/docker/install-standalone.md [hunk 12]
```diff
@@ -3,6 +3,12 @@ layout: page
 title: "Deploy Portworx with Docker"
 keywords: portworx, architecture, storage, container, cluster, install, docker, compose, systemd, plugin
 sidebar: home_sidebar
+redirect_from:
+  - /install/docker-plugin.html
+  - /docker/install-px-docker-service.html
+  - /run-with-docker.html
+  - /scheduler/docker-ucp.html
+  - /install/docker-compose.html
```
Comment (msh100) [inline scheduler/docker/install-standalone.md]: Should this redirect here?

File: scheduler/docker/install-standalone.md [hunk 13]
```diff
@@ -3,6 +3,12 @@ layout: page
 title: "Deploy Portworx with Docker"
 keywords: portworx, architecture, storage, container, cluster, install, docker, compose, systemd, plugin
 sidebar: home_sidebar
+redirect_from:
+  - /install/docker-plugin.html
+  - /docker/install-px-docker-service.html
+  - /run-with-docker.html
+  - /scheduler/docker-ucp.html
```
Comment (msh100) [inline scheduler/docker/install-standalone.md]: Should this redirect here?

File: scheduler/kubernetes/scheduler-convergence.md [hunk 14]
```diff
@@ -3,6 +3,9 @@ layout: page
 title: "Run pods on same host as a volume"
 keywords: portworx, container, Kubernetes, storage, Docker, k8s, flexvol, pv, persistent disk, StatefulSets
 sidebar: home_sidebar
+redirect_from:
+  - /kubernetes-with-scheduler-convergence.html
+  - /scheduler/kubernetes/mysql.html
```
Comment (msh100) [inline scheduler/kubernetes/scheduler-convergence.md]: https://docs.portworx.com/scheduler/kubernetes/failover.html

This is the closest page we have

File: scheduler/kubernetes/support.md [hunk 15]
```diff
@@ -3,6 +3,9 @@ layout: page
 title: "Troubleshooting PX on Kubernetes"
 keywords: portworx, container, Kubernetes, storage, Docker, k8s, flexvol, pv, persistent disk, debug, troubleshoot
 sidebar: home_sidebar
+redirect_from:
+  - /troubleshooting.html
```
Comment (msh100) [inline scheduler/kubernetes/support.md]: I don't think should forward to kubernetes support page

File: video/multi-cloud.md [hunk 16]
```diff
@@ -3,6 +3,9 @@ layout: page
 title: "Multi-cloud High Availability"
 keywords: portworx, AWS, Azure, Cloud, container, storage
 sidebar: home_sidebar
+youtubeId: es-DgA8Np0E
```
Comment (msh100) [inline video/multi-cloud.md]: Remove this line which was re-added?

File: applications/mssql-server.md (hunk 4 above)
Comment (GranSpencer09) [inline applications/mssql-server.md]: would this be a better page for the redirect? 
https://docs.portworx.com/applications/docker-registry.html

File: cloud/aws/ecs.md (hunk 5 above)
Comment (GranSpencer09) [inline cloud/aws/ecs.md]: You're thinking of redirecting this page (https://docs.portworx.com/portworx-on-ecs.html) to Amazon ECS with Portworx?

File: install/docker.md (hunk 11 above)
Comment (GranSpencer09) [inline install/docker.md]: Thank you, I'll remove this

File: install/docker.md (hunk 7 above)
Comment (GranSpencer09) [inline install/docker.md]: here? https://docs.portworx.com/knowledgebase/sdn.html

File: maintain/scale-up.md (hunk 8 above)
Comment (GranSpencer09) [inline maintain/scale-up.md]: I thought it was an old page that is no longer used, I tried to find it's another similar one

File: release-notes/portworx-enterprise.md (hunk 10 above)
Comment (GranSpencer09) [inline release-notes/portworx-enterprise.md]: I'm not certain, how does deleting the release notes impact the page?

File: scheduler/docker/install-standalone.md (hunk 13 above)
Comment (GranSpencer09) [inline scheduler/docker/install-standalone.md]: Is this a better page?
 https://docs.portworx.com/scheduler/docker/install-swarm.html

File: scheduler/docker/install-standalone.md (hunk 12 above)
Comment (GranSpencer09) [inline scheduler/docker/install-standalone.md]: Or here? 
https://docs.portworx.com/applications/docker-compose.html

File: scheduler/kubernetes/support.md (hunk 15 above)
Comment (GranSpencer09) [inline scheduler/kubernetes/support.md]: To the FAQs page instead? https://docs.portworx.com/knowledgebase/faqs.html

File: getting-started/px-enterprise.md [hunk 17]
```diff
@@ -3,6 +3,10 @@ layout: page
 sidebar: home_sidebar
 title: "Get Started with PX-Enterprise"
 keywords: portworx, px-developer, container, storage, requirements
+redirect_from:
+  - /get-started-px-enterprise.html?
+  - /get-started-px-enterprise.html
+
```
Comment (msh100) [inline getting-started/px-enterprise.md]: Please clean this up. only needs to be once and no blank line

File: video/multi-cloud.md [hunk 18]
```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Multi-cloud High Availability"
 keywords: portworx, AWS, Azure, Cloud, container, storage
 sidebar: home_sidebar
-redirect_from: "/video-multi-cloud.html"
+youtubeId: es-DgA8Np0E
```
Comment (msh100) [inline video/multi-cloud.md]: Remove the youtubeid

File: manage/volumes.md [hunk 19]
```diff
@@ -3,7 +3,11 @@ layout: page
 title: "Create and Manage Storage Volumes"
 keywords: portworx, px-enterprise, storage, volume, create volume, clone volume
 sidebar: home_sidebar
-redirect_from: "/create-manage-storage-volumes.html"
+redirect_from:
+  - /create-manage-storage-volumes.html
+  - /manage/create-and-manage.html
+  - /manage/volume-sets.html
+
```
Comment (msh100) [inline manage/volumes.md]: Remove the unnecessary blank line

File: applications/mssql-server.md (hunk 4 above)
Comment (msh100) [reply applications/mssql-server.md]: I am not too sure, no obvious pages for ddc or cloud

File: cloud/aws/ecs.md (hunk 5 above)
Comment (msh100) [reply cloud/aws/ecs.md]: I think the file should be removed if it's unreferenced for sure (and new redirect)

File: install/docker.md (hunk 7 above)
Comment (msh100) [reply install/docker.md]: I think maybe https://docs.portworx.com/scheduler/mesosphere-dcos/portworx_volumes.html - it's the closest thing mentioning Marathon

File: maintain/scale-up.md (hunk 8 above)
Comment (msh100) [reply maintain/scale-up.md]: It was actually this file https://github.com/portworx/px-docs/blob/dcf11db845f120eb11f2df503e08f7dc09c03730/run-with-docker.md

So can we move this to the https://docs.portworx.com/scheduler/docker/install-standalone.html page?

File: release-notes/portworx-enterprise.md (hunk 10 above)
Comment (msh100) [reply release-notes/portworx-enterprise.md]: Okay let's ignore this for now

File: scheduler/docker/install-standalone.md (hunk 12 above)
Comment (msh100) [reply scheduler/docker/install-standalone.md]: Yes, I think so

File: scheduler/docker/install-standalone.md (hunk 13 above)
Comment (msh100) [reply scheduler/docker/install-standalone.md]: Yeh I guess so

File: scheduler/kubernetes/support.md (hunk 15 above)
Comment (msh100) [reply scheduler/kubernetes/support.md]: Yes, seems better

**Pull request closed without merging**